### PR TITLE
Electronic Incompatibility Trait

### PR DESCRIPTION
**Pull request by ReaperofNight**
[Feature][Add to wiki][Input Wanted]

## About the PR 
This makes it so you cannot be borged by any means, or made an AI by any means, but you can still be cloned.

## Why's this needed? 
A lot of people would like to be cloned but really do not like to be borged, me included. I think this change would help people like me.

## Changelog

(u) Reaper/Drewmajor11
(+) Added electronic Incompatibility Trait(0 points.)

### CODE REVIEW

File: _std/__build.dm [hunk 1]
```diff
@@ -31,9 +31,9 @@ o+`        `-` ``..-:yooos-..----------..`
 
 //////////// OPTIONS TO GO FAST
 
-//#define IM_REALLY_IN_A_FUCKING_HURRY_HERE 1  // Skip setup for atmos, Z5, don't show changelogs, skip pregame lobby
-//#define GOTTA_GO_FAST_BUT_ZLEVELS_TOO_SLOW 1  // Only include the map Atlas, no other zlevels. Boots way faster
-//#define Z_LOG_ENABLE 1  // Enable additional world.log logging
+// #define IM_REALLY_IN_A_FUCKING_HURRY_HERE 1  // Skip setup for atmos, Z5, don't show changelogs, skip pregame lobby
+// #define GOTTA_GO_FAST_BUT_ZLEVELS_TOO_SLOW 1  // Only include the map Atlas, no other zlevels. Boots way faster
+// #define Z_LOG_ENABLE 1  // Enable additional world.log logging
```
Comment (zjdtmkhzt) [inline _std/__build.dm:36]:
```suggestion
//#define IM_REALLY_IN_A_FUCKING_HURRY_HERE 1  // Skip setup for atmos, Z5, don't show changelogs, skip pregame lobby
//#define GOTTA_GO_FAST_BUT_ZLEVELS_TOO_SLOW 1  // Only include the map Atlas, no other zlevels. Boots way faster
//#define Z_LOG_ENABLE 1  // Enable additional world.log logging
```

**Comment by BatElite** [comment]
You should briefly explain what the trait does in the changelog.

**Comment by ZeWaka** [comment]
unsure on this. discussion has good points both ways

**Comment by Adharainspace** [comment]
i think the discussion tended towards removing this, especially since borgs can commit suicide now. ill be closing this

**Pull request closed without merging**